src/components/VolumeProperties.vue

```diff
@@ -1,11 +1,17 @@
 <script lang="ts">
-import { computed, defineComponent, watch } from '@vue/composition-api';
+import { computed, defineComponent, ref, watch } from '@vue/composition-api';
 import { useCurrentImage } from '../composables/useCurrentImage';
 import { useViewConfigStore } from '../store/view-configs';
 import { CVRConfig } from '../types/views';
 
 const TARGET_VIEW_ID = '3D';
 
+const LIGHTING_MODELS = {
+  standard: 'Standard',
+  scatter: 'Scatter',
```

Comment:
Let's change "Scatter" to "Hybrid" - that is a more accurate description.

Reply:
👍️